yt_dlp/extractor/tiktok.py

```diff
@@ -276,6 +276,9 @@ def extract_addr(addr, add_meta={}):
             'like_count': int_or_none(stats_info.get('digg_count')),
             'repost_count': int_or_none(stats_info.get('share_count')),
             'comment_count': int_or_none(stats_info.get('comment_count')),
+            'download_count': int_or_none(stats_info.get('download_count')),
+            'forward_count': int_or_none(stats_info.get('forward_count')),
+            'save_count': int_or_none(stats_info.get('collect_count')),
```

Comment:
I think it's the PR-OP's burden for documenting the fields, but especially these (I'm guessing they're for reposts and favorites respectively?)

Reply:
OK, I will add all the fields back and start documenting them. I heard there are fields on the main python I might have to fill out